docs/source/user/jupyterhub.rst

```diff
@@ -0,0 +1,36 @@
+.. _jupyterlab:
+
+JupyterLab on JupyterHub
+------------------------
+
+JupyterLab works out of the box with JupyterHub, and can even run side-by-side
+with the classic Notebook. There are a couple of configuration options
+you may want to set.
+
+If you install JupyterLab on a system running JupyterHub, it will immediately be
```

Comment:
Subtitle: Set JupyterLab as default user interface

Reply:
Are you saying we should add a subheading?